aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -681,15 +681,15 @@ static Tensor &std_var_out(Tensor &result, const Tensor &self, IntArrayRef dim,
 
   if (at::isComplexType(self.scalar_type())){
     ScalarType dtype = c10::toValueType(get_dtype(result, self, {}, true));
-    Tensor real_in = self.copy_real().to(dtype);
+    Tensor real_in = at::real(self);
```

Comment:
You lost the `.to(dtype)` here, is that expected? Why was it needed before?
Same for the others below.

Reply:
`copy_real` and `copy_imag` methods returned a complex tensor (`re+0j` / `im+0j`) before where the returned tensor's imaginary part was always zero and so `.to(equivalent_float_type)` was just added to return a float tensor ( as we are doing with `real` and `imag` now)

Reply:
Sounds good!